README.md

```diff
@@ -86,10 +70,62 @@ pip install -e .[dev]
 # Configure pre-commit hooks
 pre-commit install
 ```
+### Running on Saved Restart File
+To evaluate a state obtained from a checkpoint, run Spinup-Evaluation as follows.
+
+```sh
+python main.py \
+  --restart <path-to-restart.nc> \
+  --mesh-mask <path-to-mesh_mask.nc> \
+  --output <path-to-output>
+```
+
+### Running on Predictions [TODO]
```

Comment:
The use of the code has now changed regarding #46. Also, we don't run on predictions anymore.